downstream/modules/platform/proc-upgrade-installer.adoc

```diff
@@ -71,4 +71,4 @@ $ ./setup.sh -i inventory.new.ini -e @credentials.yml -- --ask-vault-pass
 
 .Additional resources
 * For general information on using the {PlatformNameShort} installer, see the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_installation_guide/index[{PlatformName} installation guide].
-* For more information about automation mesh, see the https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_automation_mesh_guide/index[{PlatformNameShort} automation mesh guide]
+* For more information about automation mesh, see the https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html/red_hat_ansible_automation_platform_automation_mesh_guide/index[{PlatformNameShort} automation mesh guide]
```

Comment:
Rather than point to 2.1 docs from version 2.2 what about commenting the links? 

`// * For more information...`

You could add an extra comment directly preceding it like `// Remove comment when 2.2 docs are published.` as an easy to find placeholder / reminder when the rest of the 2.2 docs go out.